### ISSUE TITLE
Missing conditions in multi-allele genotype test DB

### ISSUE THREAD

**Issue by kimrutherford** [pombase/pombase-chado#472]
From Mark:

   One quick issue though, it looks like there is a large amount of
   conditions and additional data that would normally be present is missing:

   select f.uniquename, f.name, fr1t.name, f2.uniquename, f2.name, fr2t.name,
   f3.uniquename, f3.name, fcc3.name, fcpt3.name, fcp3.value from feature f
   join feature_relationship fr1 on (f.feature_id=fr1.object_id) join cvterm
   fr1t on (fr1.type_id=fr1t.cvterm_id and fr1t.name='instance_of') join
   feature f2 on (fr1.subject_id=f2.feature_id) join feature_relationship fr2
   on (f2.feature_id=fr2.subject_id) join cvterm fr2t on
   (fr2.type_id=fr2t.cvterm_id) join feature f3 on
   (fr2.object_id=f3.feature_id and fr2t.name='part_of') left join
   feature_cvterm fc3 on (f3.feature_id=fc3.feature_id) join cvterm fcc3 on
   (fc3.cvterm_id=fcc3.cvterm_id) left join feature_cvtermprop fcp3 on
   (fc3.feature_cvterm_id=fcp3.feature_cvterm_id) left join cvterm fcpt3 on
   (fcp3.type_id=fcpt3.cvterm_id) where f.uniquename like 'SPBC14C8.07c%' and
   fcc3.name like '%increased septation index%';

**Comment by kimrutherford** [comment]
Hi Mark.  What sort of thing is missing?  It doesn't look too bad compared to the v53 release.  I can't see anything obvious missing.  I'm looking at the most recent load:
  http://curation.pombase.org/dumps/builds/pombase-build-2015-08-04-v1-l1/

**Comment by markmcdowall** [comment]
Was missing the expression information, but this looks to be in there now. Worth checking in the test load once it is on staging though

**Comment by kimrutherford** [comment]
OK, great.  That was probably fixed by fe3400b7fda but I didn't think about it when you emailed.  Glad it's fixed now.  I'll check once staging is ready.

**Comment by ValWood** [comment]
added 'next' although it sounds as if it is already done

**Comment by kimrutherford** [comment]
> I have loaded the data and it is on stage.pombase.org
>
> Search for cdc18 as an example.
>
> What is strange through is the multi-allele phenotypes as none of them
>  seem to have anything for teh expression column. Does this match what is
> in the data that you load into Chado?
>
> Am I missing a link somewhere?

Hi Mark.  I've worked this out.  The expression level is now stored as a feature_relationshipprop on the relationship between an allele and a genotype.  That's between the same allele can have a different expression in different genotypes.

I checked the cdc18 alleles in your example and the correct expression levels are on the feature_relationshipprops.

https://github.com/pombase/pombase-chado/wiki/Chado-Notes#genotype-modelling

Kim.
